src/core/Akka.Remote/RemoteActorRef.cs

```diff
@@ -177,5 +178,42 @@ public override void Start()
                 Remote.Provider.UseActorOnNode(this, _props, _deploy, _parent);
         }
     }
+
+    public class LazyActorRef : IActorRef
+    {
+        private readonly Lazy<IActorRef> _lazyActorRef;
+
+        public LazyActorRef(RemoteActorRefProvider provider,Address address, string path)
+        {
+            _lazyActorRef = new Lazy<IActorRef>(() => provider.ResolveActorRefWithLocalAddress(path,address));
```

Comment:
This could have been agnostic of Akka Remote and just take the Lazy<IActorRef> instead.
But never mind that right now. I just want to see the perf impact on the experimental Remote Transport.